packages/graphql-playground-react/src/components/Playground/ResultViewer.tsx

```diff
@@ -135,7 +135,7 @@ const Result = styled<ResultProps, 'div'>('div')`
     display: none !important;
   }
   .CodeMirror-scroll {
-    overflow: auto !important;
+    overflow: scroll !important;
```

Comment:
Fixes this small glitch - not sure if it has any other side effects though...

![overflow-auto](https://user-images.githubusercontent.com/36491/78512600-a9ab4000-776b-11ea-8e7e-2134787e6f46.gif)

Reply:
hm! worth it for now. we can release all of this as an alpha or beta and see what folks say